packages/web/src/pages/sign-up-page/components/AccountHeader.tsx

```diff
@@ -1,80 +1,173 @@
-import { Avatar, Box, Flex, Text } from '@audius/harmony'
+import { Avatar, Box, Flex, IconCamera, IconImage, Text } from '@audius/harmony'
 
 import {
   getCoverPhotoField,
-  getHandleField,
   getNameField,
   getProfileImageField
 } from 'common/store/pages/signon/selectors'
 import { useMedia } from 'hooks/useMedia'
 import { useSelector } from 'utils/reducer'
 
-export const AccountHeader = () => {
+import { ImageField } from './ImageField'
+
+type AccountHeaderProps = {
+  mode: 'editing' | 'viewing'
+}
+
+const CoverPhotoBox = ({
+  imageUrl,
+  isEditing
+}: {
+  imageUrl: string | undefined
+  isEditing?: boolean
+}) => (
+  <Box
+    h='100%'
+    w='100%'
+    border='default'
+    css={{
+      backgroundColor: 'lightgray',
```

Comment:
is `lightgray` a harmony approved background color?
can we use colors from the theme here?
`const { color } = useTheme()` should provide them

Reply:
errr no it is not. This was leftover temporary junk. Removed